nms/app/fbcnms-projects/magmalte/app/theme/default.js

```diff
@@ -420,30 +420,13 @@ export default createMuiTheme({
         padding: '8px 16px',
         color: colors.primary.brightGray,
         backgroundColor: colors.primary.white,
-        '&$notchedOutline': {
-          borderColor: colors.primary.gullGray,
-        },
-        '&$hovered $notchedOutline': {
-          borderColor: colors.secondary.malibu,
-          borderWidth: '1px',
-        },
-        '&$focused $notchedOutline': {
-          borderColor: colors.secondary.dodgerBlue,
-          borderWidth: '2px',
-        },
-        '&$disabled': {
-          borderColor: colors.button.lightOutline,
-        },
       },
       multiline: {
         padding: '8px 16px',
       },
       input: {
         padding: 0,
         ...typography.button,
-        '&:read-only': {
-          opacity: 0.5,
-        },
```

Comment:
Alright so I noticed in the Dialog forms I was working in that we were declaring `readOnly` when we wanted to prevent the field from being updated.  To identify that the field should not be targetable/editable, I changed the opacity to be `50%`. What I didn't realize at the time was that `readOnly` is also defaulted for select drop-downs, hence them impacting the select you were working with. As such I've gone ahead and removed the `readOnly` overrides from the CSS, and I also updated the `readOnly` form fields to make use of the `disabled` property (see later comment)